t/05-scheduler-full.t

```diff
@@ -136,146 +136,6 @@ subtest 'Scheduler worker job allocation' => sub {
     dead_workers($schema);
 };
 
-subtest 're-scheduling and incompletion of jobs when worker rejects jobs or goes offline' => sub {
-    # avoid wasting time waiting for status updates
-    my $web_ui_connection_mock = Test::MockModule->new('OpenQA::Worker::WebUIConnection');
-    $web_ui_connection_mock->redefine(_calculate_status_update_interval => .1);
-
-    my $jobs   = $schema->resultset('Jobs');
-    my @latest = $jobs->latest_jobs;
-    shift(@latest)->auto_duplicate();
-
-    # try to allocate to previous worker and fail!
-    my $allocated = scheduler_step();
-    is @$allocated, 0, 'no jobs can be allocated to previous workers';
-
-    # simulate a worker in broken state; it will register itself but declare itself as broken
-    @workers = broken_worker(@$worker_settings, 3, 'out of order');
-    wait_for_worker($schema, 5);
-    $allocated = scheduler_step();
-    is @$allocated, 0, 'scheduler does not consider broken worker for allocating job';
-    stop_workers;
-    dead_workers($schema);
-
-    # simulate a worker in idle state that rejects all jobs assigned to it
-    @workers = rejective_worker(@$worker_settings, 3, 'rejection reason');
-    wait_for_worker($schema, 5);
```

Comment:
The tests with the broken and rejective worker should stay. These workers should behave consistently.